web/src/app/shared/models/status.model.ts

```diff
@@ -6,7 +6,7 @@ import { firestore } from 'firebase';
  */
 export class PullRequestStatusModel {
   id: number;
-  state: string;
+  state: 'open' | 'closed' | 'failure' | 'success' | 'pending';
```

Comment:
I think this is better 👍 , but seeing this now, made me think we might be able to take it a step further and create a a custom type (sorry I didn't think about this before 🤦‍♂ )...

```typescript
type state = 'open' | 'closed' | 'failure' | 'success' | 'pending';
```

http://www.typescriptlang.org/docs/handbook/advanced-types.html#string-literal-types